src/thesis.tex

```diff
@@ -634,7 +634,7 @@
     Its main roles are to:
     \begin{itemize}
         \item request and parse raw data from backend into visual representations of it,
-        \item provide an interface for the user,
+        \item provides an interface for the user,
```

Comment:
imo should be `provide` - we list responsibilities, so it is like if each `item` was prepended with `Its main roles are to`